WORKSPACE

```diff
@@ -123,9 +125,7 @@ d_repositories()
 #
 # gRPC gateway
 #
-load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
-go_rules_dependencies()
-go_register_toolchains()
+# Uses same dependencies as Go.
```

Comment:
It didn't work with duplicated calls.

Reply:
Indeed, not sure why it was repeated, since other imports already had duplicates dropped